requirements.notebook.txt

```diff
@@ -1,8 +1,8 @@
 beautifulsoup4==4.9.3
 bigquery-schema-generator==1.5
-boto3==1.19.8
-botocore<1.23.0, >=1.22.0
-cattrs==1.7.1
+boto3==1.24.59
+botocore<1.27.60, >=1.27.59
```

Comment:
just noticing, if the range is so precise (including the patch version), then it would be no different that using `==`?